test/org/traccar/FilterHandlerTest.java

```diff
@@ -3,14 +3,74 @@
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
+import org.traccar.database.IdentityManager;
+import org.traccar.model.Device;
 import org.traccar.model.Position;
 
 import java.util.Date;
 
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertNull;
 
-public class FilterHandlerTest extends BaseTest {
+public class FilterHandlerTest {
+
+    static {
+        Context.init(new IdentityManager() {
```

Comment:
Have not found better way to override `lookupAttribute` returned value than totally reinitialize `IdentityManager`